populus/compilation/backends/solc_auto.py

```diff
@@ -16,37 +16,30 @@
 from .base import (
     BaseCompilerBackend,
 )
-from .solc_combined_json import (
-    SolcCombinedJSONBackend,
-)
+
 from .solc_standard_json import (
     SolcStandardJSONBackend,
 )
 
 
-def get_solc_backend_class_for_version(solc_version):
-    if is_string(solc_version):
+def get_solc_backend_class_for_version(solc_version=None,settings=None):
```

Comment:
Get your editor so that it yells at you about this type of stuff.  PEP8 wants a space after the `,`.